src/libraries/System.Memory/src/System/Buffers/ReadOnlySequence.cs

```diff
@@ -528,6 +528,55 @@ public SequencePosition GetPosition(long offset)
             return Seek(offset);
         }
 
+        /// <summary>
+        /// Translate opaque <see cref="SequencePosition"/> value to numerical position offset
+        /// </summary>
+        /// <param name="position"></param>
+        /// <returns></returns>
+        public long GetOffset(SequencePosition position)
+        {
+            bool positionIsNotNull = position.GetObject() != null;
```

Comment:
You are calling `position.GetObject()` in another place as well(line 545) so better to store in variable and use that one.

Reply:
ohhh yep...I moved to `BoundsCheck` at the end...missed that thank's!